openapi/components/schemas/GatewayAccountConfig/MobilePay.yaml

```
@@ -8,12 +8,12 @@ allOf:
       credentials:
         type: object
         properties:
-          providerId:
+          merchantSerialNumber:
             type: string
-            description: ID of the MobilePay provider.
-          merchantVat:
+            description: Serial number of the MobilePay merchant.
+          subscriptionKey:
             type: string
-            description: VAT number of the MobilePay merchant.
+            description: Subscription key of the MobilePay.
```

nitpick: this seems to be missing text

```suggestion
            description: Subscription key of the MobilePay merchant.
```